caasp-livenessprobe-image/caasp-livenessprobe-image.kiwi

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: caasp/v4.5/livenessprobe:%%PKG_VERSION%%-rev<VERSION> caasp/v4.5/livenessprobe:%%PKG_VERSION%%-rev<VERSION>-build<RELEASE> caasp/v4.5/livenessprobe:beta -->
+
+<image schemaversion="6.9" name="caasp-livenessprobe-image" xmlns:suse_label_helper="com.suse.label_helper">
```

Comment:
Could this package name renamed to `caasp-csi-livenessprobe-image` as well, thanks.

Reply:
done